tests/test_utils/test_utils.py

```diff
@@ -9,7 +9,7 @@
 import unittest
 from shutil import rmtree, copytree
 
-import tmms.utils.utils as TmmsUtils
+import tmms.utils.core_utils as TmmsUtils
 
```

Comment:
Same whining as before.   Maybe we need a whiteboard session with art objects and all the import dependencies.  I'm lost.